lib-ltc/src/ltc.cpp

```diff
@@ -256,3 +256,142 @@ bool Ltc::ParseTimeCodeRate(const char *pTimeCodeRate, uint8_t &nFPS, type &tTyp
 
 	return true;
 }
+
+inline uint32_t _tc_round_up(double x)
+{
+	return (static_cast<uint32_t>((x + 0.5)));
+}
+
+inline uint32_t _tc_round_down(double x)
+{
+	return (static_cast<uint32_t>((x - 0.5)));
+}
+
+inline uint32_t _tc_round(double x)
+{
+	if (x < 0.0)
+		return (static_cast<uint32_t>((x - 0.5)));
+	else
+		return (static_cast<uint32_t>((x + 0.5)));
+}
+
```

Comment:
round functions should be in <math.h> [round, floor, ceil] or even better in <cmath>

Reply:
okay, these are kind of specific to my implementation, happy to move though.

Reply:
>> or even better in 
These rounds are specific to my implementation of MStoTC and TCtoMS,  maybe they be private members of Ltc class? I don't know about putting them in math.h, because I had to make them in the first place on PC because C++ math.h round didn't give reliable results.